.github/workflows/run-ui-test.yml

```diff
@@ -0,0 +1,38 @@
+name: Android Release
+
+on:
+  pull_request:
+  push:
+    branches:
+      - "develop"
+      - "main"
+
+permissions: read-all
+
+jobs:
+  apk:
+    name: Run UI tests on Firebase Test Lab
+    runs-on: ubuntu-20.04
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Prepare Android Environment
+        uses: ./.github/actions/prepare-android-env
+
+      - name: Assemble app debug APK
```

Comment:
suggestion: when you'll execute `assembleDebug` and `assembleDebugAndroidTest` in one command, you can better leverage multicore, while those two commands are executed together. I think about: `./gradlew assembleDebug assembleDebugAndroidTest`